lib/src/utils/localization/app_localizations.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+
+export 'package:flutter_gen/gen_l10n/app_localizations.dart' show L10N;
+
+class LocalizationOverride extends StatelessWidget {
```

Comment:
Seria uma boa trocar o nome desse widget

Reply:
LocalizationOverriderBuilder seria melhor?
```suggestion
class LocalizationOverriderBuilder extends StatelessWidget {
```